.devcontainer/Dockerfile

```diff
@@ -79,7 +79,7 @@ COPY ["airlock_processor/requirements.txt", "/tmp/pip-tmp/airlock_processor/"]
 RUN pip3 --disable-pip-version-check --no-cache-dir install -r /tmp/pip-tmp/requirements.txt && rm -rf /tmp/pip-tmp
 
 # Install azure-cli
-ARG AZURE_CLI_VERSION=2.37.0-1~buster
+ARG AZURE_CLI_VERSION=2.43.0-1~bullseye
```

Comment:
I actually just noticed you upgraded the cli version and not just the distro used.
The other day I fixed a situation where a cli output format has changed. Since we do a lot with AZ cli like Auth but also process its output this seems risky without testing all the flows we have.

Reply:
Want me to revert the CLI version?

Or are there some flows to test?

Reply:
I think we can leave the cli version for another time as I do expect that to break things while for the base image is less risky imo

Reply:
Done.